mne/cov.py

```diff
@@ -1431,11 +1435,11 @@ def _get_whitener_data(info, noise_cov, picks, diag=False, rank=None,
         noise_cov = cp.deepcopy(noise_cov)
         noise_cov['data'] = np.diag(np.diag(noise_cov['data']))
 
-    scalings_ = dict(mag=1e12, grad=1e11, eeg=1e5)
+    _scalings = DEFAULTS['scalings_cov_rank']
```

Comment:
Can you cherry pick, I don't like the idea of mering bugs into master. The missing copy is a bug.